src/components/NavBarNewOrder.js

```diff
@@ -0,0 +1,58 @@
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+
+export default function NavBarNewOrder (prop){
+    const navigate = useNavigate();
+
+    function logout(){
+        localStorage.removeItem(`user`);
+        navigate("/login");
+    }
+    
+    return(
+        <StyledNavBar>
+        {prop.page === "home" ?
+                <>
+                <NavBarElement>
+                <button onClick={()=> logout()}>{prop.user} logout</button>
```

Comment:
No need to create empty wrappers for functions

Reply:
If i leave it out an error is displayed.
If i replace it with a div, the style is inherited on it. What I not want to.

Reply:
done